pkg/core/manager/ai/manager.go

```diff
@@ -25,7 +25,10 @@ type AIManager struct {
 
 // NewAIManager returns a new AIManager object
 func NewAIManager(c registry.ExtraConfig) (*AIManager, error) {
-	aiClient := ai.NewClient(c.Backend)
+	if c.AIAuthToken == "" {
+		return nil, nil
```

Comment:
If the token is empty, an error should be returned, otherwise the npe error is prone to occur

Reply:
When Karpor starts, it creates a new AIManager. If return the error, the startup will fail. Is there any other way to do this?

Reply:
It should be up to the caller to decide which errors to ignore. The underlying method should ensure that the returned entity is not empty when there are no errors.